test/util/fixtures/ClientServiceFixtures.java

```diff
@@ -0,0 +1,61 @@
+package util.fixtures;
+
+import entities.Client;
+import enums.Roles;
+import enums.StatusEnum;
+import java.util.ArrayList;
+import java.util.List;
+
+public final class ClientServiceFixtures {
+
+  private static Client client1 = new Client();
```

Comment:
Remove all these instances, they are unnecessary, please use the builder instead

Reply:
solved